packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -291,25 +294,48 @@ export class ChainService implements ChainServiceInterface {
     subscriptions.map(s => s.unsubscribe());
   }
 
+  private async waitForConfirmations(event: Event | null): Promise<void> {
+    const numConfirmations = 6;
```

Comment:
This should probably be a configurable value in the constructor of the `ChainService`. Tests with ganache might want `1`, Rinkeby maybe `3`, Mainnet maybe `6`.

Reply:
We can certainly make this configurable!

I would actually argue that we should use the same value for Ganache and Rinkeby as Mainnet:
- Using a longer confirmation time might have race conditions. It would be bad to wait to discover these on Mainnet.
- Seems like it is in general a good practice to configure test environments as close as possible to production environments.

Reply:
AFAIK
- `ganache` does not mine blocks by default
- if it does mine blocks, it mines them one at a time at an interval set by some config
- there are no uncle blocks

Does our test environment have the autominer turned on?

Reply:
From the `ganache-core` readme: 

> "blockTime": number - Specify blockTime in seconds for automatic mining. If you don't specify this flag, ganache will instantly mine a new block for every transaction. Using the blockTime option is discouraged unless you have tests which require a specific mining interval.

Reply:
https://github.com/statechannels/statechannels/blob/4ab70e25e0653d41603148e1dd3b3a579d4b4799/packages/server-wallet/jest/chain-setup.ts#L37

Reply:
You can also instruct ganache to mine a block.

Reply:
I like that idea! a25bdf8